lms/urls.py

```diff
@@ -84,6 +84,9 @@
     # Course content API
     url(r'^api/course_structure/', include('course_structure_api.urls', namespace='course_structure_api')),
 
+    # Course API
+    url(r'^api/courses/', include('course_api.urls')),
```

Comment:
The folder is called course_api but the endpoint is plural `courses`. One of these needs to be changed. I think the edX API conventions has a guideline to which one should be changed.